charts/graphite/templates/_helpers.tpl

```diff
@@ -30,3 +30,10 @@ Create chart name and version as used by the chart label.
 {{- define "graphite.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
+
+{{/*
+Define the service Account name
+*/}}
+{{- define "graphite.serviceAccount.name" -}}
+{{ default "graphite" .Values.serviceAccount.accountName }}
```

Comment:
the value `accountName` is not documented. Pls also add this field to the `values.yaml` file and comment it out.